### PR TITLE
feat: email challenge improvements for v2

### PR DESCRIPTION
**Pull request by anipendakur-okta**
Updated the json mock
Changed a callout timeout value
Some properties file changes for i18n

Resolves: OKTA-296722

## PR Checklist

- [X] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot/Video:
image

### Reviewers:
@haishengwu-okta @nikhilvenkatraman-okta 

### Issue:

- [OKTA-296722](https://oktainc.atlassian.net/browse/OKTA-296722)

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -302,11 +302,13 @@ email.enroll.title = Set up Email Authentication
 email.enroll.description = Send a verification code to your registered email.
 email.link.terminal.msg = To finish signing in, return to the screen where you requested the email link.
 email.mfa.title = Verify with Email Authentication
+email.mfa.title.alternate = Verify with your email
 email.mfa.description = Send a verification code to {0}.
 email.mfa.email.sent.description = A verification code was sent to {0}. Check your email and enter the code below.
-email.mfa.email.sent.description.sentText = A verification code was sent to
```
Comment (anipendakur-okta) [inline packages/@okta/i18n/src/properties/login.properties:307]: This was only used at one place, hence changed it.

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -302,11 +302,13 @@ email.enroll.title = Set up Email Authentication
 email.enroll.description = Send a verification code to your registered email.
 email.link.terminal.msg = To finish signing in, return to the screen where you requested the email link.
 email.mfa.title = Verify with Email Authentication
+email.mfa.title.alternate = Verify with your email
```
Comment (anipendakur-okta) [inline packages/@okta/i18n/src/properties/login.properties:305]: v1 uses prop from line #304, hence added a new one for IDX

File: playground/mocks/data/idp/idx/factor-verification-email.json [hunk 3]
```diff
@@ -22,8 +22,7 @@
               "value": [
                 {
                   "name": "passcode",
-                  "label": "One-time verification code",
-                  "secret": true
```
Comment (anipendakur-okta) [inline playground/mocks/data/idp/idx/factor-verification-email.json:26]: The changes here will be done through backend - ref OKTA-303574

File: packages/@okta/i18n/src/properties/login.properties (hunk 2 above)
Comment (haishengwu-okta) [reply packages/@okta/i18n/src/properties/login.properties:305]: let's change to `oie.email.mfa.title`. fyi: https://github.com/okta/okta-signin-widget/pull/1215

File: src/v2/view-builder/views/email/RequiredFactorEmailView.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import polling from '../shared/polling';
 import BaseFactorView from '../shared/BaseFactorView';
 import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
 
-const SHOW_RESEND_TIMEOUT = 60000;
+const SHOW_RESEND_TIMEOUT = 30000;
```
Comment (haishengwu-okta) [inline src/v2/view-builder/views/email/RequiredFactorEmailView.js:9]: testcafe test `ChallengeFactorEmail_MagicLink_spec.js` need update as well.

File: packages/@okta/i18n/src/properties/login.properties (hunk 2 above)
Comment (anipendakur-okta) [reply packages/@okta/i18n/src/properties/login.properties]: updated

File: src/v2/view-builder/views/email/RequiredFactorEmailView.js (hunk 4 above)
Comment (anipendakur-okta) [reply src/v2/view-builder/views/email/RequiredFactorEmailView.js:9]: updated the test to have less timeout...

**Pull request merged**